ucsb-cs156-f24/team02-f24-00
# Pc commit index page

Closes #96 

Merge after #111 and #113

In this PR we add the commit index page.

To test:

1. Visit this dokku deployment:  https://team02-dev-pconrad.dokku-00.cs.ucsb.edu
2  Login
3. Visit the commits index page.  
4. Create and delete should work properly.

Screenshot:

<img width="1161" alt="image" src="https://github.com/user-attachments/assets/fcd48ae5-1a86-40f0-83a0-0db0a95562ba">

Storybook: https://67217d09054c99a597a2d9ac-scuarcnbeq.chromatic.com/?path=/docs/pages-commits-commitsindexpage--docs
